app/models/client.rb

```diff
@@ -33,6 +34,8 @@ def intake
     intakes && intakes.first
   end
 
+  before_create { self.last_response_at = created_at }
```

Comment:
The migration sets the default value to a "dynamic value" of CURRENT_TIMESTAMP, and per the rails docs dynamic values only get set on _existing_ columns. :( So we need to manually set it on creation for new clients. To me, it made sense to just set all three (created_at, updated_at, last_response_at) to the created_at time. 

I also considered that we could leave this value null until the client completes the intake, but dealing with null values seemed more dicey/difficult to manage. However, willing to consider a different value here.

Reply:
hm, just to make sure i understand: when you say "we need to manually set it" you're referring to this line?

Reply:
Yep, thats what's happening here!